specs/windows/windows_events.table

```diff
@@ -1,5 +1,5 @@
 table_name("windows_events")
-description("All of the Windows registry hives.")
+description("All of the Windows event logs.")
```

Comment:
While you're here, perhaps we could change this description to be `Windows Event logs.`? It seems to me a more generic description is due, as unless one has subscribed to _all_ channels, they'll only get some Windows Event logs. Mind making that change? :)